main.css

```diff
@@ -789,3 +842,53 @@ input:focus {
   align-items: center;
   justify-content: space-between;
 }
+
+/* Styling contact for desktop */
+
+@media screen and (min-width: 992px) {
+  .contact {
+    display: flex;
+    padding: 130px 142px 154px 135px;
+  }
```

Comment:
[OPTIONAL] It would be better to add `justify-content: space-around;` to the contact selector so your contact section is centered on the page.

Reply:
@medaminedev66, thanks for your comments 🙌